druid/src/lib.rs

```diff
@@ -149,6 +149,7 @@ mod data;
 mod env;
 mod event;
 mod ext_event;
+#[macro_use]
 pub mod lens;
```

Comment:
I suggest making `lens` the first module listed, because the macros are only available for the modules listed after this. Which is fine for now, but it could lead to some head-scratching down the road when the macros don't work in another module.